src/components/card/Card.js

```diff
@@ -17,17 +31,57 @@ function Card(props) {
           ) : (
             <img className="deceased" src={deceased} alt="deceased" />
           )}
+          <button
+            className="turn-page-button-front"
+            onClick={handleTurnPageButtonClick}
+          >
+            <img src={arrow} alt="Turn page" />
+          </button>
         </div>
         <div className="flip-card-back">
           <div className="flip-card-back-text">
-            <h2>{props.name}</h2>
+            <div className="emoji-button-bar">
+              <button
+                className="emoji-button"
+                onClick={() => handleEmojiButtonClick('🤓')}
+              >
+                🤓
+              </button>
```

Comment:
I would add an `aria-label` to these buttons as a screenreader would read out the strings of the emoji - I think.